### PR TITLE
Add Amazon AWS image upload to User and Badge.

### PR DESCRIPTION
**Pull request by dinocosta**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -80,3 +83,6 @@ gem 'capistrano-rails-console'
 
 # Production logging
 gem 'rails_12factor', group: :production
+
+# Environment Variables
+gem 'dotenv-rails', groups: [:development, :test]
```
Comment (frm) [inline Gemfile]: We have a `:development, :test` group [here](https://github.com/cesium/intra.seium.org/blob/6133e24b4cddf2055217a1d884725045cc6de3c7/Gemfile#L58). You can just put it together. Also, while you're at it, please fix that indentation.

**Review by frm — CHANGES_REQUESTED** [review]
The `dotenv` and new badge should be at a different PR each but oh well. Let it go.

File: Gemfile (hunk 1 above)
Comment (dinocosta) [reply Gemfile]: @frmendes Thanks for calling that out, completely missed it... Updated 👍

**Comment by dinocosta** [comment]
@frmendes Bump 😆

**Review by frm — APPROVED** [review]

**Pull request merged**